LICENSE_FAQ.md

```diff
@@ -49,12 +49,10 @@ blockchain consensus.The initial Designated Blockchains shall be the Evmos
 blockchains, identified by chain identifiers 9000 (testing network or testnet)
 and 9001 (main network or mainnet).
 
-Apart from Evmos repository, Evmos is currently building the Evmos Software
-Development Kit (Evmos SDK), to help developers to create their own
-EVM-compatible blockchain network with custom parameters. Once it is publicly
-available, you may use the Evmos SDK instead for your commercial project,
-subject to the applicable Evmos SDK license. For more information about Evmos
-SDK, check out the Evmos Manifesto.
+Apart from Evmos repository, Evmos is currently building evmOS, to help developers
```

Comment:
It's not apart from the Evmos repository atm though - all code for evmOS is within this repo